KEYCLOAK-SETUP.md

```diff
@@ -0,0 +1,55 @@
+Steps to setup Red Hat SSO:
+
+SSO Instructions:
+https://access.redhat.com/documentation/en/red-hat-single-sign-on/7.0/single/getting-started-guide/#installing_distribution_files
```

Comment:
From what I understand, the release version of Windup will use the product SSO. I do expect that it would work with the OSS Keycloak version also, though.

I'm going to try to set that one up for the unit tests today.